### PR TITLE
test(react-wrapper): add tests

### PR DESCRIPTION
**Pull request by asudoh**

### CODE REVIEW

File: tests/spec/createReactCustomElementType_spec.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  let container;
+  const events = new EventManager();
+
+  beforeEach(function() {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  describe('Attribute mapping', function() {
+    it('Should use the property name the attribute name by default', function() {
```
Comment (emyarod) [inline tests/spec/createReactCustomElementType_spec.ts]: can this be reworded? it's a bit confusing

Comment (asudoh) [reply tests/spec/createReactCustomElementType_spec.ts]: Thanks @emyarod for your thoughts! How about adding "as"...? (...use the property name as the attribute name...)

Comment (emyarod) [reply tests/spec/createReactCustomElementType_spec.ts]: yes that would be better

Comment (asudoh) [reply tests/spec/createReactCustomElementType_spec.ts]: Thanks @emyarod! Updated.

**Pull request merged**